docs/submitted_reports.md

```diff
@@ -0,0 +1,91 @@
+# Common report styles.
+
+Rageshakes can come from a number of applications, and we provide some practical notes on the generated format.
+
+At present these should not be considered absolute nor a structure to follow; but an attempt to document the
+currently visible formats as of January 2022
+
+
+## Feedback 
```

Comment:
some more context on this? is this an eleweb feature, or?

Reply:
(Will investigate)